CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change history for ui-users
 
+## [12.0.1] In Progress
+
+* Fix bug where save button is not always enabled when saving roles to multiple affiliations. Refs UIU-3331.
+* Fix detail page not immediately updated after editing user roles. Refs UIU-3333.
```

Comment:
This bug is gone now, so I don't care about it. Tell me what the code does now that it's working correctly, not about a problem you used to have, e.g. "Replace `forEach` with `map` in order to await promises and then render details".